style.css

```diff
@@ -2778,26 +2775,37 @@ ul {
   margin-right: calc(28px - 0.5em);
 }
 
+.name-and-title-badges {
```

Comment:
Should we prefix this with `community-` to stay consistent?

Reply:
Done